.ci/docker-compose-file/docker-compose-redis-single-tcp.yaml

```diff
@@ -6,8 +6,8 @@ services:
     image: public.ecr.aws/docker/library/redis:${REDIS_TAG}
     volumes:
       - ./redis/single-tcp:/usr/local/etc/redis/
-    ports:
-      - "6379:6379"
+    #ports:
+      #- "6379:6379"
```

Comment:
Leftovers?

Reply:
no, this is causing test failures in master branch